keps/349-all-or-nothing/README.md

```diff
@@ -253,11 +268,17 @@ condition, so the corresponding job is unsuspended without further waiting.
 
 ### Timeout on reaching the PodsReady condition
 
-We introduce a timeout, defined in the `waitForPodsReady.timeoutSeconds` field, on reaching the `PodsReady` condition since the job
-is unsuspended (the time of unsuspending a job is marked by the Job's
-`job.status.startTime` field). When the timeout is exceeded, the Kueue's Job
+We introduce two timeouts defined in the `waitForPodsReady.timeoutSeconds` and `waitForPodsReady.replacementTimeoutSeconds` fields.
+
+First one tracks the time between job getting unsuspended (the time of unsuspending a job is marked by the Job's
+`job.status.startTime` field) and reaching the `PodsReady=true` condition.
+
+Second one tracks the time between changing `PodsReady` condition to `false` after the job is running and reaching the
```

Comment:
How are we going to know if this is the first `PodsReady=False`? Describe how we can distinguish the two scenarios.

Reply:
Added, thanks!